WordPress/Learn
# Workshops: Adjust block template to accommodate excerpt generation

This removes all of the `core/group` blocks from the template and reassigns their custom class names to the wrapping column blocks or inner blocks where appropriate. A little bit of additional styling was necessary to ensure the layout remains the same for both the old and new markup.

With no group blocks in the mix, excerpts will now be automatically generated for workshop posts.

Fixes #46

## To test

1. First create and publish a new workshop post using the trunk branch, and fill in all the details except the manual excerpt (probably easiest to just copy [one of the posts](https://learn.wordpress.org/workshops/) on production).
1. Now checkout this PR branch and create another workshop post with the same content.
1. Now view both posts on the front end, as well as the workshop archive page. On the archive, the second post should have an auto-generated excerpt, while the first does not. On the single post views, the layout should be almost identical. The only difference should be some difference in the spacing between content blocks, but this is ok.
